app.js

```diff
@@ -14,84 +14,99 @@ if(confirmQuiz) {
+    score++;
+  } else {
+    alert('Too bad. You were wrong.');
+  }
 }
-
 console.log(score);
 //Below chooses a random number out of 100 and then has user guess that number using 'too high' and 'too low'.
-var pushupNumber = Math.floor (Math.random() * 100) + 1;
-for (var i = 0; i < 4; i++){
-  var ageGuess = prompt('How many pushups can Patrick do? You have 4 guesses.');
-  ageGuess = parseInt(ageGuess, 10);
-  console.log(pushupNumber);
-  if (ageGuess > pushupNumber){
-    alert('Too high.');
-  }else if (ageGuess < pushupNumber) {
-    alert ('Too low.');
-  }else if (ageGuess === pushupNumber){
-    alert ('You are correct.');
-    score++;
-    break;
+function questionSix() {
+  var pushupNumber = Math.floor (Math.random() * 100) + 1;
+  for (var i = 0; i < 4; i++){
+    var ageGuess = prompt('How many pushups can Patrick do? You have 4 guesses.');
+    ageGuess = parseInt(ageGuess, 10);
+    console.log(pushupNumber);
+    if (ageGuess > pushupNumber){
+      alert('Too high.');
+    }else if (ageGuess < pushupNumber) {
+      alert ('Too low.');
+    }else if (ageGuess === pushupNumber){
+      alert ('You are correct.');
+      score++;
+      break;
+    }
   }
+  alert('Patrick can do ' + pushupNumber + ' pushups! That might be really impressive or not impressive at all. The number was randomly generated!');
 }
-alert('Patrick can do ' + pushupNumber + ' pushups! That might be really impressive or not impressive at all. The number was randomly generated!');
 //Since a random number is used, number of puhhups in variable from 1 to 100
 
 //Return to this setion after you set variable at top of page into an Array.
-var myCars = ['VW Jetta', 'Subaru WRX', 'Ford Explorer'];
-var guessCounter = 1;
+function questionSeven() {
+  var myCars = ['VW Jetta', 'Subaru WRX', 'Ford Explorer'];
+  var guessCounter = 1;
 
-for (var i = 0; i < 6; i++){
-  var carGuess = prompt('Patrick has owned 3 cars. Try and guess one of them. You have 6 guesses.');
-  if( carGuess === myCars[0] || carGuess === myCars[1] || carGuess === myCars[2]){
-    alert ('Congratulations! You correctly guessed a ' + carGuess + '. That is one of the cars Patrick has owned. ');
-    score++;
-    break;
-  } else {
-    alert('No, Patrick never owned a ' + carGuess + ' This is guess ' + guessCounter + ' out of your 6 guesses.');
-    guessCounter++;
+  for (var i = 0; i < 6; i++){
+    var carGuess = prompt('Patrick has owned 3 cars. Try and guess one of them. You have 6 guesses.');
+    if( carGuess === myCars[0] || carGuess === myCars[1] || carGuess === myCars[2]){
+      alert ('Congratulations! You correctly guessed a ' + carGuess + '. That is one of the cars Patrick has owned. ');
+      score++;
+      break;
+    } else {
+      alert('No, Patrick never owned a ' + carGuess + ' This is guess ' + guessCounter + ' out of your 6 guesses.');
+      guessCounter++;
+    }
   }
+  alert ('Patrick has owned a VW Jetta, a Ford Explorer, and currently owns a Subaru WRX.');
```

Comment:
Can you think of a way to make some of the text of this draw from the answers array?